docs/static/include/pluginbody.asciidoc

```diff
@@ -910,12 +910,12 @@ Logstash plugins:
 [subs="attributes"]
 ----------------------------------
   # Gem dependencies
-  s.add_runtime_dependency 'logstash-core', '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency 'logstash-core', '>= 5.0.0', '< 6.0.0'
```

Comment:
This should be replaced by line:
`s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"`